lib/IRGen/GenConcurrency.cpp

```diff
@@ -335,15 +335,39 @@ llvm::Value *irgen::emitCreateTaskGroup(IRGenFunction &IGF,
   assert(subs.getReplacementTypes().size() == 1 &&
          "createTaskGroup should have a type substitution");
   auto resultType = subs.getReplacementTypes()[0]->getCanonicalType();
-  auto resultTypeMetadata = IGF.emitAbstractTypeMetadataRef(resultType);
+
+  // In desktop Swift, we pass a Metadata pointer as the last argument. In
```

Comment:
I know what you mean but "desktop Swift" is a bit off, iOS as well, don't we? So that's misleading now. Is this in "non-embedded"...?

Reply:
Fixed the comment.